test/msw-api/cli/init.test.ts

```diff
@@ -133,3 +133,37 @@ test('warns when current public directory does not match the saved directory fro
 
   await cleanup()
 })
+
+test('does not cause ESLint errors or warnings', async () => {
+  const { prepare, getPath, cleanup } = createTeardown(
+    './tmp/cli/init/eslint',
+    addFile('package.json', {
+      name: 'package-name',
+    }),
+    // intentionally degenerate config to cause a warning in the initial comment
+    addFile('.eslintrc.json', {
+      rules: {
+        'no-warning-comments': [
+          'warn',
```

Comment:
Do you see any implications of choosing "error" instead?

Reply:
The main thing I can think of is that warnings are considered less severe than errors by ESLint and are sometimes ignored or suppressed in reports. By using an error in our test case, the test itself could develop a bug where warnings are suppressed, causing the test to pass even though there is a warning that would be seen by an end user.